runtime/gc_ctrl.c

```diff
@@ -111,7 +111,7 @@ CAMLprim value caml_gc_counters(value v)
   /* get a copy of these before allocating anything... */
   double minwords = Caml_state->stat_minor_words
     + ((double) Wsize_bsize ((uintnat)Caml_state->young_end -
-        (uintnat) Caml_state->young_ptr)) / sizeof(value);
+                             (uintnat) Caml_state->young_ptr));
```

Comment:
Done.

Reply:
I apologise for so much discussion over one line, but in the commit I referenced above, the `/ sizeof(value)` was added in both places (erroneously in this second one), where I think the fix really should have been to use `Wsize_bsize` in `caml_gc_minor_words_unboxed` rather than `/ sizeof(value)` 🙈

Reply:
Arguably, since both `young_ptr` and `young_end` are `value *`, one could just do the subtraction, without any casts, divisions, or macro applications....